src/controllers/article.js

```diff
@@ -0,0 +1,153 @@
+import Slug from 'slug';
+import ArticleModel from '../helpers/articles';
+
+/**
+  * @description class representing Article Controller
+  * @class ArticleController
+  */
+class ArticleController {
+  /**
+    * @description - This method is responsible for creating new articles
+    * @static
+    * @param {object} request - Request sent to the router
+    * @param {object} response - Response sent from the controller
+    * @returns {object} - object representing response message
+    * @memberof ArticleController
+    */
+  static async createArticle(request, response) {
+    const {
+      title,
+      description,
+      body
+    } = request.body;
+    const userId = request.userData.payload.id;
+    try {
+      const slugGen = Slug(`${title} ${Date.now()}`);
+      const newArticle = await ArticleModel.createArticle({
+        slug: slugGen,
+        title,
+        description,
+        body,
+        userId
+      });
+      return response.status(201).json({
+        status: 'Success',
+        message: 'Article created successfully',
+        newArticle
+      });
+    } catch (error) {
+      return response.status(500).json({
+        status: 'Fail',
+        error: error.message
+      });
+    }
+  }
+
+  /**
+    * @description - This method is responsible for fetching all articles
+    * @static
+    * @param {object} request - Request sent to the router
+    * @param {object} response - Response sent from the controller
+    * @returns {object} - object representing response message
+    * @memberof ArticleController
+    */
+  static async getAllArticles(request, response) {
+    try {
+      const allArticles = await ArticleModel.getAllArticle();
+      if (!allArticles) {
+        return response.status(404).json({
+          status: 'Fail',
+          message: 'No article found'
+        });
+      }
+      return response.status(200).json({
+        status: 'Success',
+        message: 'Successfully get all articles',
```

Comment:
Nice controller to get all articles, but I can see here that you are using a future tense for your success message "Successfully get all articles". I feel it should be in a past tense since the action has already occurred, "Successfully got all articles" or what do you think?